doc/detailsDefault.md

```diff
@@ -0,0 +1,47 @@
+# Details of Default File
+
+This file stores the **id** of commands, **id** and **compName** of events, and **id** and **compName** of telemetry.
+LabVIEW Component will read this file to register commands, events and telemetry in TCP Server.
+
+The **default.ini** file is in `config/` directory.
+You can follow [here](../config/default.ini) for details.
```

Comment:
This (../config/default.ini) is the `ini` file instead of the details.

Reply:
the idea is click and go to the ini file, so I just modified the text.